src/Faker/Provider/Image.php

```diff
@@ -101,10 +120,10 @@ public static function image(
         // Generate a random filename. Use the server address so that a file
         // generated at the same time on a different server won't have a collision.
         $name = md5(uniqid(empty($_SERVER['SERVER_ADDR']) ? '' : $_SERVER['SERVER_ADDR'], true));
-        $filename = $name . '.png';
+        $filename = $name . '.' . $format;
```

Comment:
Pls use sprintf instead of string concats

Reply:
Done, may I ask why though?